### ISSUE TITLE
Broken with newer versions of Tensorflow?

### ISSUE THREAD

**Issue by Sydriax** [255BITS/HyperGAN#47]
I'm attempting to use vanilla HyperGAN, and I got hit by an error which seemed to stem from tf.concat -- "TypeError: Expected int32, got list containing Tensors of type '_Message' instead."

A bit more digging indicated that the Tensorflow API changed regarding the ordering of parameters to tf.concat, and I was able to suppress those errors by regexing through the code and reversing the order of parameters. To me, that seems to indicate that the issue is that I'm using a newer Tensorflow version than the one for which HyperGAN was written.

I've now got another set of errors that I suspect have the same root cause (Tensorflow version) -- "ValueError: Only call `softmax_cross_entropy_with_logits` with named arguments (labels=..., logits=..., ...)"

I'm presently on Tensorflow version '0.12.head'

I have no idea how much work is required to make HyperGAN work with newer versions of Tensorflow, nor am I advocating for that to happen. I do think, however, that it would be very helpful to users to indicate with which versions of Tensorflow HyperGAN is confirmed to be compatible.

Thanks!

**Comment by martyn** [comment]
Hypergan 0.8 (`develop` branch) has support for Tensorflow 1.0.

The pip package is out of date at the moment(oops :))  We'll add an explicit tensorflow requirement so others don't hit this in the future.

Once 0.8 `develop` is merged into `master` we can close this ticket.

Thanks for letting us know!

**Comment by Sydriax** [comment]
Thanks very much!

**Comment by martyn** [comment]
0.8 merged
